src/app.ts

```diff
@@ -153,17 +147,22 @@ export default class CeramicAnchorApp {
     // create connection with database
     // note that it's not active database connection
     // typeorm creates connection pools and uses them for requests
-    createConnection().then(async () => await fn()).catch((e) => {
-      logger.Err(`Failed to start Ceramic Anchor Service. Error ${e.message}`);
+    try {
+        logger.imp('Connecting to database...');
+        const connection = await createConnection();
+        logger.imp(`Connected to database: ${connection.name}`);
+        await fn()
```

Comment:
Should the call to `fn()` be outside the try/catch block?  Otherwise a failure in the `fn()` call will cause us to log that the database connection failed, even if that's not actually the cause of the failure.

Reply:
Yup. Good catch